.mailmap

```diff
@@ -72,6 +73,8 @@ Chew Choon Keat <choonkeat@gmail.com>
 Chris Andrews <cpandrews8@gmail.com> cpandrews8 <cpandrews8@gmail.com>
 Chris Johnson <chris.s.johnson4@gmail.com>
 Chris Young <chris.young@tinder.com> <chris.young@gotinder.com>
+Christian Clauss <cclauss@me.com>
```

Comment:
I am not exactly sure, but was this supposed to be a duplicate?

Reply:
We can keep line 76 and delete line 77.